lib/cloud_controller/metrics/periodic_updater.rb

```diff
@@ -45,9 +46,9 @@ def catch_error
     def update_task_stats
       running_tasks = VCAP::CloudController::TaskModel.where(state: VCAP::CloudController::TaskModel::RUNNING_STATE)
       running_task_count = running_tasks.count
-      running_task_memory = running_tasks.sum(:memory_in_mb)
-      running_task_memory = 0 if running_task_memory.nil?
-      @updaters.each { |u| u.update_task_stats(running_task_count, running_task_memory) }
+      running_task_memory = running_tasks.sum(:memory_in_mb) || 0
+      @statsd_updater.update_task_stats(running_task_count, running_task_memory)
+      @prometheus_updater.update_task_stats(running_task_count, running_task_memory * 1000 * 1000)
```

Comment:
Prometheus best practice is to [emit in base units](https://prometheus.io/docs/practices/naming/#base-units). Therefore, emitted in bytes and not MB.

Reply:
Did you find any arguments pro/contra 1000/1024?

Reply:
Not really. As you said from an academic point of view 1000 is correct, but if 1024 is used in the CF CLI, I guess we should use that for consistency.